### PR TITLE
UI-1013 Tabs Component

### PR DESCRIPTION
**Pull request by mallik-svmx**
- Tabs component refactoring
- Split component, TabNav, TabItem, TabMenu, Tab and TabContent
- Includes basic unit tests

Note: Overflow Tab Menu not complete.

### CODE REVIEW

File: lib/components/Tabs/Tab.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import TabContent from './TabContent';
+
+const propTypes = {
+  activeKey: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```
Comment (hanzge) [inline lib/components/Tabs/Tab.js]: comment is missing

File: lib/components/Tabs/Tabs.js [hunk 2]
```diff
@@ -1,238 +1,48 @@
...
-        '.slds-tabs__item.slds-active .react-slds-tab-menu button',
-        '.slds-tabs__item:hover .react-slds-tab-menu button',
-        '.slds-tabs__item .react-slds-tab-menu button:focus',
-        '{ visibility: visible }',
-      ],
-    ]);
-  }
+class Tabs extends Component {
+  state = {
+    activeKey: this.props.activeKey,
```
Comment (hanzge) [inline lib/components/Tabs/Tabs.js]: use `this.props.defaultActiveKey` to make it an uncontrolled component

File: lib/components/Tabs/Tab.js (hunk 1 above)
Comment (mallik-svmx) [reply lib/components/Tabs/Tab.js]: Updated

Comment (joshsylvester) [reply lib/components/Tabs/Tab.js]: ? don't see update.

**Comment by joshsylvester** [comment]
published. `@svmx/ui-components-lightning@0.1.31`

**Pull request merged**